packages/components/src/Progress/Progress.scss

```diff
@@ -40,7 +40,7 @@ $gradient-part-width: 100% / 6;
 		&.infinite {
 			background: transparent;
 			width: 100%;
-			animation: infinite-progression-keyframes 5s infinite;
+			animation: infinite-progression-keyframes 2s infinite;
```

Comment:
As seen with @maximenicoul frontify documentation will be updated to set the new speed time. It will be hard-coded, no need to parametrize it.